Server/Server App/Server App/Globals.cs

```diff
@@ -11,7 +11,9 @@ internal static class Globals
     {
         /* Client Username shared in the thread address space only */
         [ThreadStatic]
-        public static string client_username;
+        public static string client_username;
+
+        public static readonly object adminLoginLock = new object();
```

Comment:
No lock is needed for admin login since an admin can log in from multiple devices at the same time

Reply:
Done